src/components/App/App.js

```diff
@@ -30,15 +44,20 @@ class App extends React.Component {
           <header> 
             <h1 className="app-title">Rancid Tomatillos</h1>
           </header>
-          {/* this is where our conditional rendering will happen */}
+          {this.state.error && <h2>{this.state.error}</h2>}
           {/* selected movie within conditional */}
-          {this.state.selectedMovie ? <SelectedMovie 
+          {this.state.movies && this.state.selectedMovie ? <SelectedMovie 
                                         key={this.state.selectedMovie.id} 
                                         poster={this.state.selectedMovie.poster_path} 
                                         backdrop={this.state.selectedMovie.backdrop_path}
                                         title={this.state.selectedMovie.title}
                                         rating={this.state.selectedMovie.average_rating}
                                         releaseDate={this.state.selectedMovie.release_date}
+                                        budget={this.state.selectedMovie.budget}
```

Comment:
Good catch! I'm glad we can integrate this more specific data for our user. Maybe we can format it to look like stats - easy to read and digest, and fun to learn!